ci/ci-test-tasks/build-init.yml

```diff
@@ -1,23 +1,31 @@
 parameters:
-  - name: task
-    type: string
-  - name: BuildSourceVersion
-    type: string
+- name: BuildSourceVersion
+  type: string
 
 steps:
-- checkout: none
```

Comment:
Why did you delete the "checkout: none" step? It will checkout the current master branch and then override it with the checkout of BuildSourceVersion. Can take longer time